### PR TITLE
Correction map implemented too

### PR DESCRIPTION
**Pull request by hyunwoo18**

### CODE REVIEW

File: glideinwms/sources/factory_entries.py [hunk 1]
```diff
@@ -74,6 +75,12 @@ def acquire(self):
                     nretries=self.nretries,
                     retry_interval=self.retry_interval)
 
+                if correction_map is not None:
```
Comment (StevenCTimm) [inline glideinwms/sources/factory_entries.py:78]: Tested this code in-situ on hepcsvc03 and it worked as scheduled.  I leave the task of determining code acceptability to others but functionally it is just what I requested and needed.

**Review by StevenCTimm — APPROVED** [review]
I recommend acceptance of this pull request but would appreciate at least one other developer weighing in.  It has been tested on a real machine and real factory entries
and performs as needed.

**Comment by hyunwoo18** [comment]
What is the next step?

**Comment by hyunwoo18** [comment]
Screen Shot 2021-04-08 at 4 08 55 PM
Can I click on this "Merge pull request"? Or can only Marco do it?

**Comment by StevenCTimm** [comment]
I would like to see at least one other developer double check the code.. I believe anyone with write access can 
hit "merge pull request" but would like Marco's ok first.

**Review by jcpunk — CHANGES_REQUESTED** [review]

File: glideinwms/sources/factory_entries.py [hunk 2]
```diff
@@ -74,6 +75,12 @@ def acquire(self):
...
+                    for eachDict in condor_status.stored_data:
+                        for key, value in correction_map.items():
+                            if eachDict.get(key) is None:
+                                eachDict[key] = value
```
[the remark this answers is not in the document]
Comment (hyunwoo18) [reply glideinwms/sources/factory_entries.py:82]: Marco, Steve and I talked about your point and agreed that I would open a new ticket (https://github.com/HEPCloud/decisionengine_modules/issues/325) for adding unit tests (not just in this file but also other related files too),
as I do not have time to work on this and Steve seems to need to this correction map logic right away.

[the remark this answers is not in the document]
Comment (mambelli) [reply glideinwms/sources/factory_entries.py:82]: I know it is not a good practice but this time we'll address the test in a separate ticket, #325

**Review by mambelli — APPROVED** [review]

**Comment by mambelli** [comment]
Forced merge. Unit test will be added in separate ticket #325

**Pull request merged**